index.bs

```diff
@@ -2360,53 +2372,60 @@ The entry key MUST be the extension identifier, and the value MUST be the [=clie
     });
 </pre>
 
-Extension definitions MUST specify the valid values for their client argument. Clients SHOULD ignore extensions with an invalid
-client argument. If an extension does not require any parameters from the [RP], it SHOULD be defined as taking a Boolean client
+Extension definitions MUST specify the valid values for their [=client extension input=]. Clients SHOULD ignore extensions with a invalid
+[=client extension input=]. If an extension does not require any parameters from the [RP], it SHOULD be defined as taking a Boolean client
 argument, set to `true` to signify that the extension is requested by the [RP].
 
-Extensions that only affect client processing need not specify an authenticator argument. Extensions that affect authenticator
-processing MUST specify a method of computing the authenticator argument from the client argument. For extensions that do not
-require additional parameters, and are defined as taking a Boolean client argument set to `true`, this method SHOULD consist of
-passing an authenticator argument of `true` (CBOR major type 7, value 21).
+Extensions that only affect client processing need not specify [=authenticator extension input=]. Extensions that have authenticator
+processing MUST specify the method of computing the [=authenticator extension input=] from the [=client extension input=]. For extensions that do not
+require input parameters and are defined as taking a Boolean [=client extension input=] value set to `true`, this method SHOULD consist of
+passing an [=authenticator extension input=] value of `true` (CBOR major type 7, value 21).
 
 Note: Extensions should aim to define authenticator arguments that are as small as possible. Some authenticators communicate
     over low-bandwidth links such as Bluetooth Low-Energy or NFC.
 
 
-## Extending <dfn>client processing</dfn> ## {#extension-client-processing}
+## <dfn>Client extension processing</dfn> ## {#client-extension-processing}
```

Comment:
there's a couple problems with the change in this line:
1) it removes the dfn for "client processing", thus occurrences of `[=client processing=]` are dangling, and, 
2) `<dfn>Client extension processing</dfn>` and `{#client-extension-processing}` both cause bikeshed to render html elements with ids of "client-extension-processing"